apps/richie/templates/services/nginx/configs/richie.conf.j2

```diff
@@ -84,5 +84,11 @@ server {
     add_header Cache-Control public;
     root /data/static;
     try_files /$file =404;
+
+    add_header Access-Control-Allow-Origin https://{{ richie_host }};
```

Comment:
Should we enforce the `https` protocol? This breaks HTTP compatibility.

Reply:
I thought it was already the case in our deployments. Where can I get this info from variables?

Reply:
As far as I know we have no variable containing which schema to use. But all our routes are configured to use `https` I think.

Reply:
From a route perspective, you are right: https://github.com/openfun/arnold/blob/master/apps/richie/templates/services/nginx/route.yml.j2#L22 I was focused on the nginx configuration.